lib/command.js

```diff
@@ -39,10 +39,18 @@ var commands = require('../commands');
  * @public
  */
 function Command(name, args, options, callback) {
+  if (typeof options === 'undefined') {
+    options = {};
+  }
   this.name = name;
-  this.replyEncoding = options && options.replyEncoding;
-  this.errorStack = options && options.errorStack;
+  this.replyEncoding = options.replyEncoding;
+  this.errorStack = options.errorStack;
   this.args = args ? _.flatten(args) : [];
+  if (options.keyPrefix) {
```

Comment:
should be:

var keyPrefix = options.keyPrefix;
if (keyPrefix) {
  this._iterateKeys(function (key) {
    return keyPrefix + key;
  });
}

Otherwise each iteration it will look for a pointer from options to .keyPrefix and we want to avoid it